### ISSUE TITLE
Nalopen: forrmulierrichtlijnen en “voldoen aan WCAG”

### ISSUE THREAD

**Issue by hidde** [nl-design-system/documentatie#542]
In de richtlijnen gebruiken we op sommige plekken deze constructie: 

> Door `X`, voldoe je aan WCAG-criterium `Y`

Soms is daar een opsomming van criteria.

Ik denk dat heel fijn is om zulke concrete handvaten te bieden aan lezers, lijkt me informatie waar men naar op zoek kan zijn of die kan helpen.

Maar op de huidige manier is het wmb net te ongenuanceerd: 

- in bijna alle gevallen kun je naast `X` goed doen ook nog wel iets anders fout doen, waardoor je niet meer voldoet aan het criterium
- in een opsomming is het te onduidelijk welke actie leidt tot voldoen aan welk criterium

Voorstel: laten we dit nalopen en in de meeste gevallen vervangen met iets als “`X` is nodig om te voldoen aan WCAG-criterium `Y`”. Denk dat we dan in de meeste gevallen kunnen garanderen dat dat in ieder geval zo is.

**Comment by hidde** [comment]
lossen we op in https://github.com/nl-design-system/documentatie/issues/562

**Comment by rianrietveld** [comment]
In plaats van 

> Door het geven van een logische volgorde van informatie voldoe je aan het WCAG-succescriterium [1.3.2 Betekenisvolle volgorde](https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html) (niveau A).

Zeggen:

> Het geven van een logische volgorde van informatie is nodig om te voldoen aan het WCAG-succescriterium [1.3.2 Betekenisvolle volgorde](https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html) (niveau A).

Dus:
dus de toevoeging **nodig om te** erbij.

Zie discussie in Slack:
https://codefornl.slack.com/archives/C02CWGL2M18/p1706161264610819

**Comment by rianrietveld** [comment]
PR op #587

**Comment by rianrietveld** [comment]
gefixt in  #587
